gems/quilt_rails/README.md

```diff
@@ -434,3 +434,28 @@ const app = createServer({
 
 export default app;
 ```
+
+### Fixing rejected CSRF tokens for new user sessions
+
+If a React component calls back to a Rails endpoint (e.g., `/graphql`), Rails may throw a `Can't verify CSRF token authenticity` exception. This stems from the Rails CSRF tokens not persisting until after the first `UiController` call ends.
+
+To fix this:
+
+- Add an `X-Shopify-Server-Side-Rendered: 1` header to all server-side GraphQL requests
```

Comment:
Once we have the work @ismail-syed  is doing merged it will be relatively easy for us to add this header automagically I think. Until then this makes sense 👍